API.ts

```diff
@@ -11,6 +11,7 @@ const publicURL = "https://public-api.secure.pixiv.net/"
 
 export default class API {
     private readonly userAgent = {"user-agent": "PixivIOSApp/7.7.5 (iOS 13.2.0; iPhone XR)"}
+    private readonly referer = {"referer": "https://www.pixiv.net/en/"}
```

Comment:
I'd probably just concatenate this with the user agent and call it headers. So full variable would look like:
`private readonly headers = {"user-agent": "PixivIOSApp/7.7.5 (iOS 13.2.0; iPhone XR)", "referer": "https://www.pixiv.net/"}`
I don't think the /en/ is necessarily needed here they just check if the referer is pixiv.net.